src/src/Helpers/Modbus_mgr.cpp

```diff
@@ -155,8 +176,52 @@ bool ModbusMGR_struct::disconnect(uint8_t deviceID) {
       return true; // Successfully disconnected
     }
   }
+  # ifdef MODBUS_DEBUG
+  addLogMove(LOG_LEVEL_INFO, log);
+  # endif // ifdef MODBUS_DEBUG
+  return true;
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+void ModbusMGR_struct::dumpAdminInfo()
+{
+  addLogMove(LOG_LEVEL_INFO, F("Modbus_mgr: Dumping admin info"));
+  #ifdef MODBUS_DEBUG
+  // Iterate over the modbus links and dump their info
+  for (int i = 0; i < MAX_MODBUS_LINKS; i++) {
+    if (_modbus_links[i] != nullptr)  {
+      addLogMove(LOG_LEVEL_INFO,
+                 strformat(F("Modbus_mgr: Link[%d] Port=%s, RX=%d, TX=%d, Baudrate=%d, DerePin=%d, RS485Mode=%s, CollisionDetect=%s"),
+                           i,
+                           ESPEasySerialPort_toString(_modbus_links[i]->port),
+                           _modbus_links[i]->serial_rx,
+                           _modbus_links[i]->serial_tx,
+                           _modbus_links[i]->baudrate,
+                           _modbus_links[i]->dere_pin,
+                           _modbus_links[i]->rs485_mode ? F("Yes") : F("No"),
+                           _modbus_links[i]->collision_detect ? F("Yes") : F("No")
+                           ));
+    }
+    else {
+      addLogMove(LOG_LEVEL_INFO, strformat(F("Modbus_mgr: Link[%d] <not used>"), i));
+    }
+  }
 
-  return false; // TODO: implement
+  // Iterate over the modbus devices and dump their info
+  for (int i = 0; i < MAX_MODBUS_DEVICES; i++) {
+    if (_modbus_devices[i] != nullptr)  {
+      addLogMove(LOG_LEVEL_INFO,
+                 strformat(F("Modbus_mgr: Device[%d] DeviceID=%d, LinkPort=%s"),
+                           i,
+                           _modbus_devices[i]->deviceID,
+                           ESPEasySerialPort_toString(_modbus_devices[i]->link->port)
+                           ));
+    }
+    else {
+      addLogMove(LOG_LEVEL_INFO, strformat(F("Modbus_mgr: Device[%d] <not used>"), i));
+    }
+  }
+  #endif // MODBUS_DEBUG
 }
 
-#endif // FEAURE_MODBUS
+#endif // FEATURE_MODBUS_FAC
```

Comment:
If you leave that comment out, and reformat using Uncrustify, it will be set according to the `#if` automagically (but has to be redone when changing the `#if`, though)